### DISCUSSION TITLE
Big updates: Sharing NAR Decoders and Add AdaptiveLayerNorm

### DISCUSSION THREAD

**Discussion by lifeiteng** [lifeiteng/valle#21] [Announcements]
PR https://github.com/lifeiteng/vall-e/pull/18
1. Sharing Non-AR Decoders
2. Implement AdaptiveLayerNorm and Custom Transformer Layers
3. Add `Inference` results `https://github.com/lifeiteng/vall-e/tree/main/egs/libritts/infer`

**Comment by jry-king** [comment]
Are these results synthesized by `test.sh`?

**Reply by lifeiteng** [reply]
> Are these results synthesized by `test.sh`?

in https://github.com/lifeiteng/vall-e/tree/main/egs/libritts/infer
